Dockerfile

```
@@ -1,4 +1,4 @@
-FROM node:22.9-bookworm AS frontend-builder
+FROM node:22.10-bookworm AS frontend-builder
```

https://hub.docker.com/layers/library/node/22.10.0-bookworm/images/sha256-b9996f9bf9c8de527a1572e44f8f3284dab41b386ba18cb7f4c154b532f60555?context=explore

```suggestion
FROM node:22.10.0-bookworm AS frontend-builder
```